This patch suppresses error messages when trying to wrap native plugin:
*** NSPlugin Viewer  *** ERROR: /usr/lib64/browser-plugins/opensc-signer.so: wrong ELF class: ELFCLASS64
https://bugzilla.novell.com/show_bug.cgi?id=400194
================================================================================
--- src/npw-config.c
+++ src/npw-config.c
@@ -538,6 +538,10 @@
 	  if (pid < 0)
 		continue;
 	  else if (pid == 0) {
+		if (!g_verbose)
+		{
+			fclose(stderr);
+		}
 		execl(viewer_path, NPW_VIEWER, "--test", "--plugin", filename, NULL);
 		exit(1);
 	  }
